frontend/app/components/ToolCard.tsx

```diff
@@ -119,6 +119,14 @@ export default function ToolCard (props: ToolCardProps) {
           aria-describedby={buttonLoadingId}
           aria-busy={updateToolNavLoading}
         >
+          <Button
+            onClick={() => setShowMoreInfo(!showMoreInfo)}
+            aria-expanded={showMoreInfo}
+            aria-label={`Show ${moreOrLessText} Info`}
+            startIcon={!showMoreInfo ? <ExpandMoreIcon /> : <ExpandLessIcon />}
+          >
+            {moreOrLessText}
+          </Button>
```

Comment:
Minor detail: The Mock up now say `Learn more`  change reflect `More`. don't know if we are sticking with existing design from CAE as discussed in the Slack yesterday. Keeping the changes to minimal. I am ok if that's the case

Reply:
I will keep the "More" vs "Less" labels for now. Thanks!

Reply:
I'd have to look at the notes but I know we had a lot of UX back and forth about this button. I also believe it had an accessibility issue and I'm not sure if it was resolved. I'll have to see it when it's running, but that would be a separate issue.